Sources/Automerge/patch.swift

```diff
@@ -19,12 +27,19 @@ public struct Patch: Equatable {
     }
 }
 
+/// The type of change to apply along with the data to change.
```

Comment:
I think my comment above applies here as well and to the doc comments below. The patch is a notification that something has happened, not something a user creates to make something happen. On that basis I think the documentation should use the passive past tense rather than imperative present.

Reply:
I really prefer to avoid any usage of passive voice, primarily because interpreting it can leave readers very muddy about what's taking the action. I'll definitely switch up the language, but I think we're far better off keeping it active voice, even if we use past tense to describe it.